basic_camera/console.cpp

```diff
@@ -1,17 +1,45 @@
+#include "pch.h"
 #include "console.h"
+#include "video.h"
 
-console::console()
-{
+
+console::console():
+	run_loop(true)
+{	
 }
 
 console::~console()
 {
 }
 
-void console::take_pic()
-{
-}
+void console::console_screen()
+{	
+	video.open_cam();
+	while (run_loop)
+	{
+		std::cout << "Please enter filename and saving format -for example file.jpg- when you want to take a photo. To exit, please enter 'q' \n";
+		std::string input;
+		std::getline(std::cin, input);
 
-void console::choose_type()
-{
+		if (!(input != "q" ^ input != ""))
```

Comment:
Simplify this

Reply:
Actually I put this like that because when you enter this if statement, you will face second if which is `if (input.find(".")!=std::string::npos)` and if you enter nothing as a input, this instruction throws an exception. That's why I added this.